mlir/utils/jenkins/Dockerfile

```diff
@@ -86,19 +99,15 @@ RUN groupadd -g 109 render
 RUN wget --no-check-certificate -qO - http://repo.radeon.com/rocm/rocm.gpg.key 2>/dev/null | apt-key add -
 RUN echo "deb [arch=amd64] $ROCM_DEB_REPO $ROCM_BUILD_NAME $ROCM_BUILD_NUM" > /etc/apt/sources.list.d/rocm.list
 
-# Install cmake distribution
-RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | \
-    tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null
-RUN echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ focal main' | \
-    tee /etc/apt/sources.list.d/kitware.list >/dev/null
-
 # Note instead of installing the latest, we should always manually bump cmake version when necessary.
 # This make sure that we don't accidentally use newer cmake features incompatible with our client.
-# As of 2022-08-17, the rocm/rock images has /opt/conda/bin/cmake at 3.22.1, so we'll match that.
-# Note that the mlir minimum is 3.18.
-RUN apt-get update && rm /usr/share/keyrings/kitware-archive-keyring.gpg && \
+# llvm-premerge-checks installs 3.23.3, see above.
+# As of 2022-08-17, the rocm/rock images has /opt/conda/bin/cmake at 3.22.1.
```

Comment:
I don't know which image has /opt/conda so I can't verify that version.  external/llvm-project/mlir/CMakeLists.txt does have 3.20.0.